### ISSUE TITLE
Allow format string to be set in handler

### ISSUE THREAD

**Issue by bendavis78** [adafruit/Adafruit_CircuitPython_Logging#59]
Right now the log handler has a fixed format:

```python
    def format(self, record: LogRecord) -> str:
        """Generate a timestamped message.

        :param record: The record (message object) to be logged
        """

        return f"{record.created:<0.3f}: {record.levelname} - {record.msg}"
```

The only way to customize this format is to subclass the handler. It would be more convienient to just be able to set the format string at the time the handler is initialized, eg:

```python
log_handler = StreamHandler(format_str="[%(asctime)s] %(levelname)s: %(message)s")
```

Better yet, it would be nice to just mimic the way python does formatters:

```python
formatter = logging.Formatter(fmt="[%(asctime)s] %(levelname)s: %(message)s")
log_handler = StreamHandler(fmt=formatter)
```

**Comment by FoamyGuy** [comment]
#61 adds support for Formatter objects and `setFormatter()` on the Handler class.

**Comment by FoamyGuy** [comment]
resolved by #61
